network-api/networkapi/templates/pages/buyersguide/about/press.html

```diff
@@ -10,200 +10,222 @@ <h1 class="tw-h2-heading">
     </h1>
 
     <p>
-        {% blocktrans trimmed %}
-            <em>*Privacy Not Included</em> helps consumers shop for trustworthy connected products. But it also helps journalists
-            cover connected products. Our research assists reporters in incorporating product privacy—and not just price
-            and performance—into their consumer technology coverage.
-        {% endblocktrans %}
-    </p>
-    <p>
-        {% blocktrans with name="Kevin Zawacki" email="kevin@mozillafoundation.org" trimmed %}
-            If you’re a member of the press interested in speaking with Mozilla’s <em>*Privacy Not Included</em> team, please email
-            {{ name}} at <a href="mailto:{{ email }}">{{email }}</a>. Our researchers can answer questions about
-            specific products, or speak to broader trends in the realm of privacy, security, and trustworthy AI.
-            We can also provide high-resolution graphics and gifs from our guide.
+        {% blocktrans with best_of_url="/privacynotincluded/categories/best-of/" trimmed %}
```

Comment:
I don't like that this is a hardcoded URL but I guess it works for this page :grimacing:

Reply:
I have the same concern